plugins/woocommerce/src/Blocks/BlockTypes/Reviews/ProductReviewsPagination.php

```diff
@@ -32,12 +32,22 @@ protected function render( $attributes, $content, $block ) {
 		}
 
 		$classes            = ( isset( $attributes['style']['elements']['link']['color']['text'] ) ) ? 'has-link-color' : '';
-		$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classes ) );
+		$wrapper_attributes = get_block_wrapper_attributes(
+			array(
+				'class'               => $classes,
+				'data-wp-interactive' => 'woocommerce/blockified-product-reviews',
+			)
+		);
+
+		$p = new \WP_HTML_Tag_Processor( $content );
+		while ( $p->next_tag( 'a' ) ) {
+			$p->set_attribute( 'data-wp-on--click', 'actions.navigate' );
+		}
```

Comment:
What's the benefit of modifying the anchors here instead of doing so in `ProductReviewsPaginationNumbers`?

Reply:
Because we also have next/prev link in `ProductReviewsPaginationNext` and `ProductReviewsPaginationPrevious` classes.